From 08ccd79319e66b3f0b51fd5f5b539e652a16dfc0 Mon Sep 17 00:00:00 2001
From: Mark Asselstine <mark.asselstine@windriver.com>
Date: Wed, 30 Apr 2014 11:06:12 -0400
Subject: [PATCH] automake: ensure VPATH builds work correctly

There were a few non-VPATH friendly directives in some Makefile.am files, fix
these up to allow VPATH builds to work.

Signed-off-by: Mark Asselstine <mark.asselstine@windriver.com>
---
 config/etc/Makefile.am | 4 ++--
 src/tests/Makefile.am  | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/config/etc/Makefile.am b/config/etc/Makefile.am
index 81d7709..dca001e 100644
--- a/config/etc/Makefile.am
+++ b/config/etc/Makefile.am
@@ -4,8 +4,8 @@ distroconf = @LXC_DISTRO_CONF@
 
 EXTRA_DIST = default.conf.ubuntu default.conf.libvirt default.conf.unknown
 
-default.conf:
-	cp $(distroconf) $@
+default.conf: $(distroconf)
+	cp $? $@
 
 clean-local:
 	@$(RM) -f default.conf
diff --git a/src/tests/Makefile.am b/src/tests/Makefile.am
index 7f6645b..641e207 100644
--- a/src/tests/Makefile.am
+++ b/src/tests/Makefile.am
@@ -64,7 +64,7 @@ buildtest-TESTS: $(TESTS)
 install-ptest:
 	install -d $(TEST_DIR)
 	install -D ../lxc/liblxc.so $(TEST_DIR)/../lxc/liblxc.so
-	install -D ../../config/test-driver $(TEST_DIR)/../../config/test-driver
+	install -D $(top_srcdir)/config/test-driver $(TEST_DIR)/../../config/test-driver
 	cp Makefile $(TEST_DIR)
 	@(for file in $(TESTS); do install $$file $(TEST_DIR);  done;)
 	sed -i 's|^Makefile:|_Makefile:|' $(TEST_DIR)/Makefile
-- 
1.8.3.2

